template/prompts.md

```
@@ -1 +1,28 @@
+
+Crea una **página web con lógica en javascript que invierta el orden de una cadena de texto**.
+
+Ejemplo: si introduzco `mañana es halloween` devuelve `neewollah se anañam`.
+
+Hazlo apoyado en el seed `index.html` que proporcionamos dentro de la carpeta template
+
+# template
+
+Encontraras adjunto una imagen con el template definido para el diseño de la pagina, este no debe ser modificado.
+
+#index.html
```

markdownlint-cli2 (0.18.1)

No space after hash on atx style heading

(MD018, no-missing-space-atx)